diff -ur xmonad-contrib-0.9.1/XMonad/Hooks/DynamicLog.hs new/XMonad/Hooks/DynamicLog.hs
--- xmonad-contrib-0.9.1/XMonad/Hooks/DynamicLog.hs	2009-12-17 00:32:25.000000000 +0100
+++ new/XMonad/Hooks/DynamicLog.hs	2010-08-12 09:40:11.267160695 +0200
@@ -253,7 +253,7 @@
     -- run extra loggers, ignoring any that generate errors.
     extras <- sequence $ map (flip catchX (return Nothing)) $ ppExtras pp
 
-    return $ encodeOutput . sepBy (ppSep pp) . ppOrder pp $
+    return $ sepBy (ppSep pp) . ppOrder pp $
                         [ ws
                         , ppLayout pp ld
                         , ppTitle  pp wt
