--- xmonad-contrib-0.10/XMonad/Hooks/DynamicLog.hs	2011-11-19 00:26:06.000000000 +0100
+++ new/XMonad/Hooks/DynamicLog.hs	2011-12-02 04:23:11.130463658 +0100
@@ -276,7 +276,7 @@
     -- run extra loggers, ignoring any that generate errors.
     extras <- mapM (flip catchX (return Nothing)) $ ppExtras pp
 
-    return $ encodeString . sepBy (ppSep pp) . ppOrder pp $
+    return $ sepBy (ppSep pp) . ppOrder pp $
                         [ ws
                         , ppLayout pp ld
                         , ppTitle  pp wt
