diff -u -r in/mplayer-1.0_rc4_p20111215/mplayer.c out/mplayer-1.0_rc4_p20111215/mplayer.c
--- in/mplayer-1.0_rc4_p20111215/mplayer.c	2011-12-15 13:22:11.000000000 +0100
+++ out/mplayer-1.0_rc4_p20111215/mplayer.c	2012-01-01 10:52:04.000000000 +0100
@@ -1886,7 +1886,8 @@
         // finally select subs by language and container hints
         if (dvdsub_id == -1 && dvdsub_lang)
             dvdsub_id = demuxer_sub_track_by_lang(mpctx->demuxer, dvdsub_lang);
-        if (dvdsub_id == -1)
+        // load default subtitle, unless the user doesn't want that
+        if (dvdsub_id == -1 && sub_auto)
             dvdsub_id = demuxer_default_sub_track(mpctx->demuxer);
         if (dvdsub_id >= 0) {
             id    = dvdsub_id;
