diff --git a/engine/tabdict.py b/engine/tabdict.py
index 368734d..90f1975 100644
--- a/engine/tabdict.py
+++ b/engine/tabdict.py
@@ -45,7 +45,8 @@ tab_dict = {
     'R':76, 'S':77, 'T':78, 'U':79, 'V':80,
     'W':81, 'X':82, 'Y':83, 'Z':84, '0':85,
     '1':86, '2':87, '3':88, '4':89, '5':90,
-    '6':91, '7':92, '8':93, '9':94
+    '6':91, '7':92, '8':93, '9':94,
+    "ß":95, "ẞ":96, "ł":97, "Ł":98
     }
 
 tab_key_list = tab_dict.keys()
