--- data/Makefile	2010-02-10 22:17:25.000000000 +0100
+++ data/Makefile.new	2010-03-02 15:29:31.047890420 +0100
@@ -1,10 +1,10 @@
 all: build
 
 ICON = im-twitter.png
-ICON_DIR = /usr/share/icons/hicolor/48x48/hildon
+ICON_DIR = $(DESTDIR)/usr/share/icons/hicolor/48x48/hildon
 
 PROFILE = twitter.profile
-PROFILE_DIR = /usr/share/osso-rtcom
+PROFILE_DIR = $(DESTDIR)/usr/share/osso-rtcom
 
 build clean:
 	
