diff --git a/src/gtkprpltwtr/gtkprpltwtr.c b/src/gtkprpltwtr/gtkprpltwtr.c
index 710f3da..4cd167f 100644
--- a/src/gtkprpltwtr/gtkprpltwtr.c
+++ b/src/gtkprpltwtr/gtkprpltwtr.c
@@ -307,6 +307,10 @@ static gboolean twitter_uri_handler(const char *proto, const char *cmd_arg, GHas
             purple_debug_error(PLUGIN_ID, "malformed uri. Invalid id for reply\n");
             return FALSE;
         }
+
+        // TODO we should test if they're a friend and only then suppress it, but meh...
+        return TRUE;
+        
         conv = twitter_endpoint_reply_conversation_new(twitter_endpoint_im_find(account, TWITTER_IM_TYPE_AT_MSG), user, id, TRUE);
         if (!conv) {
             return FALSE;
